src/components/FormPairRow.tsx

```diff
@@ -30,7 +25,7 @@ export interface IPairRow {
 interface IMultiTag {
   isVerified: boolean;
   isLST: boolean;
-  isUnknown: boolean;
+  // isUnknown: boolean;
```

Comment:
Can just remove this?

Reply:
we're still experimenting what's the more approachable and understandable for our users